mlir/test/python/ir/operation.py

```diff
@@ -82,12 +82,16 @@ def walk_operations(indent, op):
     # CHECK:           OP 1: func.return
     walk_operations("", op)
 
-    # CHECK:    Region iter: <mlir.{{.+}}.RegionIterator
-    # CHECK:     Block iter: <mlir.{{.+}}.BlockIterator
-    # CHECK: Operation iter: <mlir.{{.+}}.OperationIterator
-    print("   Region iter:", iter(op.regions))
-    print("    Block iter:", iter(op.regions[-1]))
-    print("Operation iter:", iter(op.regions[-1].blocks[-1]))
+    # CHECK:    Regions: [<mlir.{{.+}}.Region object at {{.+}}>]
+    # CHECK:     Blocks: [<mlir.{{.+}}.Block object at {{.+}}>]
+    # CHECK: Operations: [<mlir.{{.+}}.FuncOp object at {{.+}}>]
+    print("   Regions:", [*op.regions])
+    print("    Blocks:", [*op.regions[-1]])
+    print("Operations:", [*op.regions[-1].blocks[-1]])
```

Comment:
i get this fine but others might not (a little too "pythonic"); prefer

```suggestion
    print("   Regions:", list(op.regions))
    print("    Blocks:", list(op.regions[-1]))
    print("Operations:", list(op.regions[-1].blocks[-1]))
```